docs_src/OVMS/pipelinesetup.md

```diff
@@ -0,0 +1,86 @@
+# Set up Pipeline
+
+### Step 1: Clone the repository
+
+```
+git clone  https://github.com/intel-retail/automated-self-checkout.git && cd ./automated-self-checkout
+```
+
+### Step 2: Build the benchmark images (coming up)
```

Comment:
I prefer to call out the Docker images as there is also picture `image` we are downloading as part of steps.

```suggestion
### Step 2: Build the benchmark Docker images (coming up)
```